packages/botonic-plugin-flow-builder/src/functions/conditional-queue-status.ts

```diff
@@ -7,14 +7,24 @@ type ConditionalQueueStatusArgs = {
   request: ActionRequest
   queue_id: string
   queue_name: string
+  check_available_agents: boolean
+}
+
+enum QueueStatusResult {
+  open = 'open',
```

Comment:
don't we use upper case normally for enum key?
```
enum QueueStatusResult {
  OPEN = 'open',
  ...
}
```

Reply:
yes you are right

Reply:
I will apply it to follow the same as we have throughout the code, even though the lint rule indicates a warn

![Captura de pantalla 2024-03-04 a las 12 01 58](https://github.com/hubtype/botonic/assets/36898236/9adac2e2-b999-440a-846d-45497815425a)